packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -266,16 +277,11 @@ export class ExecutionEngineHttp implements IExecutionEngine {
    */
   async getPayload(payloadId: PayloadId): Promise<allForks.ExecutionPayload> {
     const method = "engine_getPayloadV1";
-    const executionPayloadRpc = await this.rpc.fetchWithRetries<
-      EngineApiRpcReturnTypes[typeof method],
-      EngineApiRpcParamTypes[typeof method]
-    >(
-      {
-        method,
-        params: [payloadId],
-      },
-      getPayloadOpts
-    );
+    const executionPayloadRpc = await (this.jobQueue.push({
+      method,
+      params: [payloadId],
+      methodOpts: getPayloadOpts,
+    }) as Promise<EngineApiRpcReturnTypes[typeof method]>);
```

Comment:
getPayload doesn't need to be sequential right? Don't feel it needs to be queued

Reply:
right, can keep it out of the queue, but ideally there shouldn't be anything else in the queue at that moment anyways. still can keep it out.